### PR TITLE
CB-8317: Make it work to load about:blank and to dispatch exit message

### PR DESCRIPTION
**Pull request by shingotoda**
JIRA: https://issues.apache.org/jira/browse/CB-8317

Please take a look at my patch and give feedback. I'm not really confident on the modification. As alternative workaround, I thought isCurrentlyLoading should be set to true just after loadUrlNow() if an app is running on KitKat.

### CODE REVIEW

File: framework/src/org/apache/cordova/CordovaWebViewClient.java [hunk 1]
```diff
@@ -171,8 +171,13 @@ public void onPageStarted(WebView view, String url, Bitmap favicon) {
     public void onPageFinished(WebView view, String url) {
         super.onPageFinished(view, url);
         // Ignore excessive calls.
-        if (!isCurrentlyLoading) {
-            return;
+        //     On KitKat, loading about:blank does not invoke WebViewClient#onPageStarted
+        //     So do not check isCurrentLoading if app is running on KitKat
+        //         https://code.google.com/p/android/issues/detail?id=65701
+        if (android.os.Build.VERSION.SDK_INT != android.os.Build.VERSION_CODES.KITKAT){
```
Comment (agrieve) [inline framework/src/org/apache/cordova/CordovaWebViewClient.java]: Have you confirmed that it's working on Lollipop? If not, we should probably just remove the isCurrentlyLoading guard.

Comment (shingotoda) [inline framework/src/org/apache/cordova/CordovaWebViewClient.java]: I just tested it on Lollipop and confirmed onPageStarted was NOT called for about:blank loading so my patch does not work on Lollipop. The same bug is still there in Android > KITKAT.

Comment (agrieve) [inline framework/src/org/apache/cordova/CordovaWebViewClient.java]: Hmm, saw your comment about maybe setting it in loadUrlNow(). Wondering if maybe the best thing to do is just ignore the check when url.startsWith("about:")? WDYT? e.g.:

    if (!isCurrentlyLoading && !url.startsWith("about:")) {
        return;
    }

Comment (shingotoda) [inline framework/src/org/apache/cordova/CordovaWebViewClient.java]: Ah, I agree with you on that. It looks the best thing. That worked fine as well.

Comment (shingotoda) [inline framework/src/org/apache/cordova/CordovaWebViewClient.java]: So do I need to resend pull request again with revised patch?

Comment (agrieve) [inline framework/src/org/apache/cordova/CordovaWebViewClient.java]: You don't need to re-request a PR, but you do need to make the change an upload it to github (either as a separate commit, or by force-pushing an ammended commit). Then, add a comment to let me know you've done so and I'll pull it in (once you do it, it should show up on this interface).

Comment (shingotoda) [inline framework/src/org/apache/cordova/CordovaWebViewClient.java]: Thank you. I've uploaded it. Is it OK?

**Pull request closed without merging**